driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -572,26 +615,7 @@ public void onSet(Connection connection, Message.Response response, long latency
                         if (retry == null)
```

Comment:
I wonder if we should do something more in the default case of the switch block above: currently we only increment a counter and release the connection.